libexec/bats-core/bats-exec-suite

```diff
@@ -23,6 +23,7 @@ while [[ "$#" -ne 0 ]]; do
     num_jobs="$1"
     ;;
   -x)
+    # shellcheck disable=SC2034
     extended_syntax_flag='-x'
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2034

Leaving this unused variable for backwards compatibility.